joomla/joomla-cms
# Display quickicons only for active components

Pull Request for Issue #40684 .
When components are disabled, they don't show up in the components menu, but in quick icos. 
This PR hides also quickicons for disabled components.

### Summary of Changes
Check if a component is enabled before rendering a quicicon.


### Testing Instructions
see #40684

Activate quickicons for unprotected components ( tags, banners, contacts, newsfeeds, finder, workflows, associations)
Check if they are displayed. 
Then disable these components.
Check again - the quickicons ar no longer displayed.
If you re-activates the components, they come back.


### Actual result BEFORE applying this Pull Request
Quickicons to disabled components are displayed


### Expected result AFTER applying this Pull Request
Quickicons to disabled components are not displayed


### Link to documentations
Please select:
